frontend/fingertips-frontend/components/views/OneIndicatorTwoOrMoreAreasView/OneIndicatorTwoOrMoreAreasView.test.tsx

```diff
@@ -80,23 +81,56 @@ describe('OneIndicatorTwoOrMoreAreasView', () => {
 
       expect(
         mockIndicatorsApi.getHealthDataForAnIndicator
-      ).toHaveBeenCalledTimes(expectedAreaCodes.length);
-
-      expectedAreaCodes.forEach((areaCode, index) => {
-        expect(
-          mockIndicatorsApi.getHealthDataForAnIndicator
-        ).toHaveBeenNthCalledWith(
-          index + 1,
-          {
-            areaCodes: [areaCode],
-            indicatorId: Number(testIndicators),
-          },
-          API_CACHE_CONFIG
-        );
-      });
+      ).toHaveBeenNthCalledWith(
+        1,
+        {
+          areaCodes: expectedAreaCodes,
+          indicatorId: Number(testIndicators),
+        },
+        API_CACHE_CONFIG
+      );
     }
   );
 
+  it('should make appropriate number of calls to the healthIndicatorApi with the expected parameters with a long list of areas', async () => {
+    const testIndicators = '1';
+    const testAreas = new Array(15).fill('a', 0, 15);
+    const testGroup = 'G001';
+    const expectedAreaCodes = [...new Array(10).fill('a', 0, 10), 'G001'];
+
+    const searchState: SearchStateParams = {
+      [SearchParams.IndicatorsSelected]: [testIndicators],
+      [SearchParams.GroupSelected]: testGroup,
+    };
+    mockIndicatorsApi.getHealthDataForAnIndicator.mockResolvedValueOnce([]);
+
+    await OneIndicatorTwoOrMoreAreasView({
+      searchState: searchState,
+      areaCodes: testAreas,
+    });
+
+    expect(mockIndicatorsApi.getHealthDataForAnIndicator).toHaveBeenCalledTimes(
+      Math.ceil(expectedAreaCodes.length / maxIndicatorAPIRequestSize)
+    );
```

Comment:
Can't this just be `2`?
```suggestion
    expect(mockIndicatorsApi.getHealthDataForAnIndicator).toHaveBeenCalledTimes(2);
```